.github/workflows/ci.yml

```diff
@@ -146,7 +146,7 @@ jobs:
     needs: [typecheck, audit]
     strategy:
       matrix:
-        py-version: [ {semantic: '3.9', tox: 'py39'}, {semantic: '3.12', tox: 'py312'} ]
+        py-version: [ {semantic: '3.10', tox: 'py310'}, {semantic: '3.12', tox: 'py312'} ]
```

Comment:
do we want to add 3.11 here too now? now that its only 3 versions seems strange to be stingy ehre and use only 2